pkg/openstack/loadbalancer_test.go

```diff
@@ -945,3 +946,106 @@ func Test_getBoolFromServiceAnnotation(t *testing.T) {
 		})
 	}
 }
+
+func TestLbaasV2_getMemberSubnetID(t *testing.T) {
+	lbaas := &LbaasV2{
+		LoadBalancer: LoadBalancer{
+			opts: LoadBalancerOpts{
+				LBClasses:      map[string]*LBClass{"lbclassKey": {MemberSubnetID: "lb-class-member-subnet-id-5678"}},
+				MemberSubnetID: "lb-class-memberSubnetId",
+			},
+		},
+	}
+
+	type fields struct {
+		lbaas LbaasV2
+	}
```

Comment:
Okay, I think I now understand why this is put here, there are two cases where we need the LbaasV2 struct and use the one defined above. It's quite a bit hard to read. Could we instead extend testcase structure with `opts LoadBalancerOpts` and define the options required in each case? Then we can initialize the LbaasV2 object with these options in the loop itself. I think it'll be clearer.